### DISCUSSION TITLE
Data Card es divergente de BrandFactory

### DISCUSSION THREAD

**Discussion by BrunoBiagioniNeto** [Telefonica/mistica-design#1118] [Q&A] [answered]
Una duda, Figma y Brandfactory son diferentes con sus configuraciones de uso.

No tenemos la misma configuración de uso para Figma.

![image](https://user-images.githubusercontent.com/123083886/224112590-c036834e-7781-4930-a262-437a141c0ec6.png)

**Comment by aweell** [comment, marked as answer]
Buenas @BrunoBiagioniNeto,

Esa issue se resuelve con https://github.com/Telefonica/mistica-design/issues/1075

Esta closed y saldrá en la release [8.0.0](https://github.com/Telefonica/mistica-design/milestone/17)
